port/osx/omrosbacktrace_impl.c

```diff
@@ -42,6 +42,7 @@
 
 #include "omrintrospect.h"
 
+#if !defined(OMR_ARCH_AARCH64)
```

Comment:
I would have preferred that the condition here matched the condition that enables use of these functions, that is `defined(OMR_ARCH_X86)` instead of `!defined(OMR_ARCH_AARCH64)`, but given those are the only two architectures on which macOS is supported, the conditions are equivalent for all practical purposes.

Reply:
I am going to implement the backtrace feature for AArch64 macOS sometime later.
I would like this PR to be merged soon.